src/checkers/style.css

```diff
@@ -1,4 +1,4 @@
-body {
+/*body {
```

Comment:
The inclusion of that CSS was breaking the new view CSS. Should I change the old css to use a class-based approach?